default/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+var rfdc = require('../index.js')
+
+module.exports = clone
+
+function clone (o) {
+  return rfdc()(o)
```

Comment:
It's probably best to use an already "instantiated" cloning function instead of creating a new one each time.

```suggestion
module.exports = rfdc()
```

Reply:
But isn't the name `clone` in `import clone from 'rfdc/default'` bound to whatever we export? 
I thought that doing it as you suggest would result in `import rfdc from 'rfdc/default'`

Reply:
@NearW yes, the exported function is going to have `clone` as name